consensus/polybft/consensus_runtime.go

```diff
@@ -477,6 +463,13 @@ func (c *consensusRuntime) restartEpoch(header *types.Header) error {
 	c.lock.Lock()
 	c.epoch = epoch
 	c.lastBuiltBlock = header
+
+	proposerSnapshot := c.proposerSnapshot.Copy()
+	if err := c.updateProposerSnapshot(proposerSnapshot, header.Number); err != nil {
+		c.logger.Warn("Could not update proposer calculator", "err", err)
```

Comment:
Nitpick, just to be consistent with other logs.

```suggestion
		c.logger.Warn("could not update proposer calculator", "err", err)
```